graphene_sqlalchemy/tests/test_batching.py

```diff
@@ -73,6 +73,40 @@ def resolve_reporters(self, info):
     return graphene.Schema(query=Query)
 
 
+def get_full_relay_schema():
+    class ReporterType(SQLAlchemyObjectType):
+        class Meta:
+            model = Reporter
+            name = "Reporter"
+            interfaces = (relay.Node,)
+            batching = True
+            connection_class = Connection
+
+    class ArticleType(SQLAlchemyObjectType):
+        class Meta:
+            model = Article
+            name = "Article"
+            interfaces = (relay.Node,)
+            batching = True
+            connection_class = Connection
+
+    class ReaderType(SQLAlchemyObjectType):
+        class Meta:
+            model = Reader
+            name = "Reader"
+            interfaces = (relay.Node,)
+            batching = True
+            connection_class = Connection
+
+    class Query(graphene.ObjectType):
```

Comment:
As mentioned, the original test schema defines custom resolve_ methods which I find odd, but maybe people use it like that. We use it like this.